### PR TITLE
Dreamview:add sensormeasurement info in simworld object at backend

### PR DESCRIPTION
**Pull request by lj0919**
modify some small points

### CODE REVIEW

File: modules/dreamview/frontend/proto_bundle/sim_world_proto_bundle.json [hunk 1]
```diff
@@ -475,6 +475,15 @@
                 }
               }
             },
+            "SensorMeasurements": {
```
Comment (vlin17) [inline modules/dreamview/frontend/proto_bundle/sim_world_proto_bundle.json:478]: This is frontend related. Can we exclude it in this PR?

File: modules/dreamview/proto/simulation_world.proto [hunk 2]
```diff
@@ -277,4 +281,7 @@ message SimulationWorld {
 
   // A map from a story name to whether it is on
   map<string, bool> stories = 28;
+
+  // A map from a sensor_id to a group of sensormeasurements
```
Comment (vlin17) [inline modules/dreamview/proto/simulation_world.proto:285]: s/sensormeasurements/sensor_measurements/

File: modules/dreamview/backend/simulation_world/simulation_world_service.cc [hunk 3]
```diff
@@ -600,6 +617,19 @@ void SimulationWorldService::SetObstaclePolygon(
...
+void SimulationWorldService::SetObstacleSensorMeasurements(
+    const PerceptionObstacle &obstacle, Object *world_object) {
+  if (world_object == nullptr) {
+    return;
+  }
+  for (const auto &sensor : obstacle.measurements()) {
+    Object *obj = (*(world_.mutable_sensor_measurements()))[sensor.sensor_id()]
+                      .add_sensor_measurement();
+    CreateWorldObjectFromSensorMeasurement(sensor, obj);
+    AINFO << obj->DebugString();
```
Comment (unacao) [inline modules/dreamview/backend/simulation_world/simulation_world_service.cc:629]: remove this log

File: modules/dreamview/frontend/proto_bundle/sim_world_proto_bundle.json (hunk 1 above)
Comment (lj0919) [reply modules/dreamview/frontend/proto_bundle/sim_world_proto_bundle.json:478]: 收到

File: modules/dreamview/backend/simulation_world/simulation_world_service.cc (hunk 3 above)
Comment (lj0919) [reply modules/dreamview/backend/simulation_world/simulation_world_service.cc:629]: 收到

**Pull request merged**